JH_RestAPI/settings.py

```diff
@@ -123,8 +123,13 @@
     'JH_RestAPI.backends.ModelBackend',
 )
 
-SOCIAL_AUTH_LINKEDIN_OAUTH2_KEY = os.environ['JOBHAX_LINKEDIN_CLIENT_KEY']
-SOCIAL_AUTH_LINKEDIN_OAUTH2_SECRET = os.environ['JOBHAX_LINKEDIN_CLIENT_SECRET']
+try:
+    SOCIAL_AUTH_LINKEDIN_OAUTH2_KEY = os.environ['JOBHAX_LINKEDIN_CLIENT_KEY']
+    SOCIAL_AUTH_LINKEDIN_OAUTH2_SECRET = os.environ['JOBHAX_LINKEDIN_CLIENT_SECRET']
```

Comment:
You can do `os.environ.get('JOBHAX_LINKEDIN_CLIENT_KEY', '')` instead. It's more pythonic.

Reply:
Thanks man! Looking forward you in our Slack!